ssm_jax/linear_gaussian_ssm/demos/blocked_gibbs_lgssm.ipynb

```diff
@@ -0,0 +1,1261 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    plt.plot(smoothed_posterior.smoothed_means[:,:,0].T, alpha=0.05, color='blue')
</pre><p>the posterior samples do not seem reasonable - not symmetric around the truth, and maybe too wide?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/ssm-jax/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>